core/nuxeo-connection.js

```diff
@@ -163,6 +172,12 @@ import './nuxeo-element.js';
           };
         }
       } else {
+        if (this.token && this.token !== '') {
```

Comment:
`this.token` with and empty string is false so there's no need for the  ` && this.token !== ''` part.